src/graphics/vulkan/vulkan_graphics.cpp

```
@@ -479,7 +479,42 @@ namespace moth_graphics::graphics::vulkan {
         if (context == nullptr) {
             return;
         }
+
+        // Letterbox: fit the logical aspect inside the physical extent and
+        // centre it. Bars outside the viewport stay black via the render pass
+        // clear.
+        VkExtent2D const physical = context->m_target->GetVkExtent();
+        float const logicalAspect = static_cast<float>(logicalSize.x) / static_cast<float>(logicalSize.y);
```

Clang (14.0.6)

[error] variable 'logicalAspect' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)